ios/sdk/sdk.xcodeproj/project.pbxproj

```diff
@@ -138,6 +144,7 @@
 		0BD906E71EC0C00300C8C18E /* src */ = {
 			isa = PBXGroup;
 			children = (
+				C69EFA02209A0EFD0027712B /* callkit */,
```

Comment:
Now that callkit/ exists, we can package everything together (like invite/) i.e. I'll move CallKit.m inside it as well.